### PR TITLE
[tools] Add git statistics for merged pull request

### PR DESCRIPTION
**Pull request by haoxli**
- Rename workstat to gitstat
- Add git statistics for merged pull request

https://crosswalk-project.org/jira/browse/XWALK-6119

### CODE REVIEW

**Comment by haoxli** [comment]
Pending to merge, need to fix one bug.

**Comment by haoxli** [comment]
git_stat.py only count merged commit from since date to until date, not contain pull request commit.
git_stat_pr.py count the commit which merged by pull request from since date to until date.

File: tools/gitstat/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Difference
+
+Becuase commit log ordered are different when using pull pequest strategy
+and directl git commit strategy, there are two different scripts:
+
+- **git_stat.py**: only count merged commit from since date to until date,
+  not contain merged pull request commit.
+- **git_stat_pr.py**: count the commit which merged by pull request
+  from since date to until date.
```
Comment (Honry) [inline tools/gitstat/README.md]: 's/until date/due date'

File: tools/gitstat/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  -a, --author       optional, Specify merged commit author
+
+  -o, --output       optional, Specify an output file for statistics
+
+
+## Difference
+
+Becuase commit log ordered are different when using pull pequest strategy
+and directl git commit strategy, there are two different scripts:
```
Comment (Honry) [inline tools/gitstat/README.md]: direct1?

**Comment by Honry** [comment]
LGTM

**Pull request merged**